src/apps/competitions/tasks.py

```diff
@@ -646,6 +646,22 @@ def do_phase_migrations():
         p.check_future_phase_submissions()
 
 
+@app.task(queue='site-worker', soft_time_limit=60 * 5)
+def manual_migration(phase_id):
+    try:
```

Comment:
think we should combine these into the same try except block? and just say _a_ phase was missing, rather than an error telling us which one?

Reply:
I think this is OK as is, could be useful to know where it's messing up in logs later